examples/next-app/components/ExampleComponent.tsx

```diff
@@ -15,11 +15,13 @@ import {
 } from '@/components/Select';
 import { Waveform } from '@/components/Waveform';
 
-function getTop3Expressions(expressionOutputs: EmotionScores) {
+function getTop3Expressions(
+  expressionOutputs: Hume.empathicVoice.EmotionScores,
+) {
   return Object.entries(expressionOutputs)
     .sort((a, b) => b[1] - a[1])
     .slice(0, 3)
-    .map(([key, value]) => ({ name: key, score: value }));
+    .map(([key, value]) => ({ name: key, score: value as number }));
```

Comment:
is this type assertion necessary? ideally, `number` would be inferrable from `EmotionScores`

Reply:
Previously `EmotionScores` was typed as `z.record()` and when you did `Object.entries()` typescript would be able to infer the value type. However, now that `EmotionScores` is an object with properties, you lose the typing of the values of `Object.entries`

Reply:
I'm pretty sure we can write bit of code that will do the type narrowing for us, but since this code is in the example nextjs app and not the core SDK package, I'm fine with moving forward with this and fixing it in a follow-up